From: Christoph Hellwig <hch@infradead.org>
Date: Sun, 17 Jun 2012 18:40:51 -0400
Subject: [PATCH] target: remove dead SCF_ flags
Git-commit: e1306bdab3af8bef620990a99e613f99eb30065c
References: FATE#313550
Patch-Mainline: v3.6

Remove the dead SCF_SE_ALLOW_EOO and SCF_DELAYED_CMD_FROM_SAM_ATTR
from se_cmd_flags_table.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Nicholas Bellinger <nab@linux-iscsi.org>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/target/target_core_transport.c |    1 -
 include/target/target_core_base.h      |    2 --
 2 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/target/target_core_transport.c b/drivers/target/target_core_transport.c
index cf42b2e..b602717 100644
--- a/drivers/target/target_core_transport.c
+++ b/drivers/target/target_core_transport.c
@@ -1933,7 +1933,6 @@ static void target_execute_cmd(struct se_cmd *cmd)
 
 	if (atomic_read(&dev->dev_ordered_sync) != 0) {
 		spin_lock(&dev->delayed_cmd_lock);
-		cmd->se_cmd_flags |= SCF_DELAYED_CMD_FROM_SAM_ATTR;
 		list_add_tail(&cmd->se_delayed_node, &dev->delayed_cmd_list);
 		spin_unlock(&dev->delayed_cmd_lock);
 
diff --git a/include/target/target_core_base.h b/include/target/target_core_base.h
index 15039dd..1ae97fe 100644
--- a/include/target/target_core_base.h
+++ b/include/target/target_core_base.h
@@ -166,14 +166,12 @@ enum se_cmd_flags_table {
 	SCF_SCSI_RESERVATION_CONFLICT	= 0x00000040,
 	SCF_FUA				= 0x00000080,
 	SCF_SE_LUN_CMD			= 0x00000100,
-	SCF_SE_ALLOW_EOO		= 0x00000200,
 	SCF_BIDI			= 0x00000400,
 	SCF_SENT_CHECK_CONDITION	= 0x00000800,
 	SCF_OVERFLOW_BIT		= 0x00001000,
 	SCF_UNDERFLOW_BIT		= 0x00002000,
 	SCF_SENT_DELAYED_TAS		= 0x00004000,
 	SCF_ALUA_NON_OPTIMIZED		= 0x00008000,
-	SCF_DELAYED_CMD_FROM_SAM_ATTR	= 0x00010000,
 	SCF_PASSTHROUGH_SG_TO_MEM_NOALLOC = 0x00020000,
 	SCF_ACK_KREF			= 0x00040000,
 };
-- 
1.7.4.2

